src/test/resources/serviceHearingValuesForControllerTest.json

```diff
@@ -0,0 +1,94 @@
+{
+  "caseName":null,
+  "caseNamePublic":"Test Case Name",
+  "autoListFlag":false,
+  "hearingType":"BBA3-substantive",
+  "caseType":null,
+  "caseCategories":[
+    {
+      "categoryType":"caseType"
+    },
+    {
+      "categoryType":"caseSubType"
+    }
+  ],
+  "duration":30,
+  "hearingPriorityType":"normal",
+  "numberOfPhysicalAttendees":0,
+  "hearingInWelshFlag":false,
+  "caseAdditionalSecurityFlag":false,
+  "facilitiesRequired":[
+
```

Comment:
minor: can we get rid of these trailing newlines please? same comment applies to others in this file.

Reply:
removed trailing new lines